### ISSUE TITLE
Add a flag for rows expanded from a rowspan row

### ISSUE THREAD

**Issue by void285** [gambolputty/wikitable2csv#11]
Hi, I found the tool will automatically repeat cells for specific times if the original row has rowspan attribute, and if the row have cells which have colspan attribute, the cells content will only be outputed in the first assocated row, I wonder if this is in design or a bug.

And for tables with rowspan and colspan cells, I have to do some manual combine/clean work sometimes, so if there is a flag to indicate the row is expanded from a colspan and/or a rowspan row, it will help a lot. 

Thank you for this  good tool, it really helps!

**Comment by gambolputty** [comment]
This is by design. But as you pointed out, it is not a good solution :(
The flag is a good idea, though

**Comment by gambolputty** [comment]
Do you still have to do some extra work when rows/cols are expanded after the recent update? Would a flag indicating expanded rows/cols still be helpful? Or what about an extra option to turn expanding on/off before submitting the form?

So many questions. Thanks in advance!

**Comment by void285** [comment]
Yes, I think this function will be usefull for people. Please consider the situation in the following code: the last-second column of row1-row3 are same, and the three words seprated by "、" should be in row1 / row2 / row3 sepratedly.  With a flag, users can always easy find and edit them.

A flag in regular pattern will help here and there, and can be easily deleted if people don't need it. So please add it. I think the idea about an extra option is wonderfull. :) 

```
| 太祖 | 升天皇帝           | 耶律亿 | 阿保机 | 872年-926年 | 神册、天赞、天显 | 907年-926年 |
| 太祖 | 大圣大明天皇帝     | 耶律亿 | 阿保机 | 872年-926年 | 神册、天赞、天显 | 907年-926年 |
| 太祖 | 大圣大明神烈天皇帝 | 耶律亿 | 阿保机 | 872年-926年 | 神册、天赞、天显 | 907年-926年 |
|      | 贞烈皇后           | 述律平 | 月理朵 | 879年-953年 | 天显             | 926年-927年 |
|      | 淳钦皇后           | 述律平 | 月理朵 | 879年-953年 | 天显             | 926年-927年 |
```

I also have an idea about the falg: put a tag in <\d(r|c)_s> format at the end of the grid, for example <2r_s> means rowspan=2, and <3c_s> means colspan=3.

```
| 神册、天赞、天显<3r_s> |
| 神册、天赞、天显       |
| 神册、天赞、天显       |
| 天显<2r_s>             |
| 天显                   |
```

**Comment by void285** [comment]
Hi, I just found this chrome extension https://merribithouse.net/copytables/, it helps copy tables and adds **\num** to indicate rowspan and **/num** to indicate colspan.
![image](https://user-images.githubusercontent.com/612912/43356340-8ba7145e-92a1-11e8-916b-e4dd6ea4177e.png)

**Comment by gambolputty** [comment]
I'll look into this and see if I can indicate expanded rows/cols in the next release.

**Comment by void285** [comment]
Hi, I just found the grammar I mentioned above is the Textile standard, here is two pages about it, for your infomation. :)
http://redcloth.org/hobix.com/textile/
http://redcloth.org/hobix.com/textile/quick.html
